sdk/communication/azure-communication-chat/src/test/java/com/azure/android/communication/chat/ChatThreadAsyncClientTest.java

```diff
@@ -885,7 +885,8 @@ public void canListReadReceiptsWithOptions(HttpClient httpClient) throws Executi
         assertNotNull(readReceiptList.get(0).getSenderCommunicationIdentifier());
     }
 
-    @ParameterizedTest
+    // REVISIT: Unreliable test
+    // @ParameterizedTest
     @MethodSource("com.azure.android.core.test.TestBase#getHttpClients")
```

Comment:
we can leave the `@ParameterizedTest` as it its and add `@Disabled("<reason for suppression>")`, like this: https://github.com/Azure/azure-sdk-for-android/blob/8c0b5d1a4283d58dd7cbd747a2987630b4827778/sdk/core/azure-core-http-httpurlconnection/src/test/java/com/azure/android/core/http/httpurlconnection/implementation/RestProxyWithHttpUrlConnectionTests.java#L44